backend/apis/utils/validation-utils.js

```diff
@@ -68,6 +71,13 @@ function _validDate(propertyName) {
       : `Property ${propertyName} must be a string date of format YYYY-MM-DD. Received '${val}'`;
 }
 
+function _validTime(propertyName) {
+  return val =>
+    /^[0-1][0-9]:[0-5][0-9]:[0-5][0-9]$/.test(val) && !isNaN(new Time(val))
```

Comment:
I don't think that `Time` is a real function. `new Time()` throws an error when I run it.

Reply:
Would just removing the  "&& !isNaN(new Time(val))" be enough?

Reply:
No. `Time` is not a thing in javascript at all. You should just get rid of the `&&` clause altogether because that's just a straight up bug because you're trying to call a function that doesn't exist.

<img width="302" alt="image" src="https://user-images.githubusercontent.com/5524384/63206484-6c762e80-c072-11e9-99eb-aa0b1c96d3d9.png">